package.json

```diff
@@ -1,7 +1,28 @@
 {
-  "dependencies": {},
-  "devDependencies": {},
+  "name": "wunder-d8-starter",
+  "version": "1.0.0",
+  "description": "",
+  "keywords": [
+    "Drupal 8",
+    "Eslint",
+    "Stylelint"
+  ],
+  "author": "Wunder",
+  "license": "MIT",
+  "dependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-plugin-import": "^2.20.1",
+    "stylelint": "^13.2.0",
+    "stylelint-config-prettier": "^7.0.0",
+    "stylelint-config-standard": "^20.0.0"
+  },
   "scripts": {
-    "build": "echo 'No build command specified in package.json'"
```

Comment:
We should leave the `build` command since it is used by Silta's `npm-install-build` [command](https://github.com/wunderio/silta-circleci/blob/77e66eaa5882cb6417dec2691f5a38d1126d3765/orb/commands/%40npm.yml#L9) (and probably others).